raven/base.py

```diff
@@ -636,7 +637,10 @@ def capture(self, event_type, data=None, date=None, time_spent=None,
             **kwargs)
 
         # should this event be sampled?
-        if self._random.random() < self.sample_rate:
+        if sample_rate is None:
```

Comment:
i dont think we need type checking -- people can deal with the consequence of putting arbitrary values in